example/otel-collector/main.go

```diff
@@ -76,17 +77,17 @@ func initProvider() (*otlp.Exporter, *push.Controller) {
 	global.SetMeterProvider(pusher.Provider())
 	pusher.Start()
 
-	return exp, pusher
+	return func() {
+		bsp.Shutdown() // shutdown the processor
+		pusher.Stop()  // pushes any last exports to the receiver
```

Comment:
This probably also needs a call to `exp.Shutdown(context.Background())`

Reply:
Thanks for your comment. I am wondering if there is any order in shutting down exporter and processor? I read the doc, but could not find anything.

Reply:
That's a really good question.  My initial thought is that we'd want to shut down the processor first to ensure that its queue is drained and then shut down the exporter to ensure that any remaining spans are exported.

Reply:
Updated. Please have a look. Thanks.